src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -137,7 +153,14 @@ void initializeReference() {
      */
     void initializeReads() {
         if (! readArguments.getReadFiles().isEmpty()) {
-            SamReaderFactory factory = SamReaderFactory.makeDefault().validationStringency(readArguments.getReadValidationStringency());
+            // Prefetcher is useful for cloud files because of the latencies involved.
+            boolean gsPathPresent = readArguments.getReadPaths().stream()
+                .filter(p->p.toUri().getScheme().equals("gs")).findAny().isPresent();
```

Comment:
What happens if there's a mix of GCS and non-GCS inputs? Can we avoid wrapping the non-GCS inputs? I think this wrapping would be better done in `ReadsDataSource`, which could explicitly call `SamInputResource.of(path, wrapper)` as appropriate on each input.